src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.Helix.targets

```diff
@@ -246,9 +255,9 @@
       <Output TaskParameter="ProcessedItems" ItemName="HelixProcessedWorkItemForList"/>
     </RemoveItemMetadata>
 
-    <WriteItemsToJson JsonFileName="$(TestListFilename)" Items="@(HelixProcessedWorkItemForList)" ForceJsonArray="true" />
+    <WriteItemsToJson JsonFileName="$(HelixLogFolder)$(TestListFilename)" Items="@(HelixProcessedWorkItemForList)" ForceJsonArray="true" />
```

Comment:
Unrelated to your change, but isn't the above redundant (denoted with `  <!-- Redundant  code -->`?

```
     <ItemGroup>
       <HelixProcessedWorkItem>
         <Command Condition="'$(UseScriptRunner)'!='true'">%(Command)</Command>
         <!-- When UseScriptRunner is set, we'll wrap commands provided with special Helix-y goo for telemetry-->
         <Command Condition="'$(UseScriptRunner)'=='true' AND '$(TargetsWindows)' == 'true'">$(HelixPythonPath) $(RunnerScript) --script %(Command)</Command>
         <Command Condition="'$(UseScriptRunner)'=='true' AND '$(TargetsWindows)' != 'true'">chmod +x $HELIX_WORKITEM_PAYLOAD/*.sh &amp;&amp; $(HelixPythonPath) $(RunnerScript) --script %(Command)</Command>
         <CorrelationPayloadUris>[$(CorrelationPayloadUris)]</CorrelationPayloadUris>
         <PayloadUri>$(DropUri)%(RelativeBlobPath)$(DropUriReadOnlyToken)</PayloadUri>
         <WorkItemId>%(WorkItemId)</WorkItemId>   <!-- Redundant  code -->
         <TimeoutInSeconds>%(TimeoutInSeconds)</TimeoutInSeconds>   <!-- Redundant  code -->
       </HelixProcessedWorkItem>
     </ItemGroup>
```